syntax = "proto3";
import "google/protobuf/timestamp.proto";

message LicenseEvent {
    enum Action {
        CREATED = 0;
        UPDATED = 1;
        DELETED = 2;
        ACTIVATED = 3;
    }
    string type_name = 1;
    string name = 2;
    string description = 3;
    string type = 4;
    string subdivision_id = 5;
    string tenant_id = 6;
    string status = 7;
    int32 count_requests = 8;
    google.protobuf.Timestamp activated = 9;
    google.protobuf.Timestamp expirated = 10;
    google.protobuf.Timestamp created = 11;
    google.protobuf.Timestamp expiration = 12;
    string id = 13;
    Action action = 14;
}
